### PR TITLE
mate: update packages

### PR DESCRIPTION
**Pull request by romildo**
###### Motivation for this change

- Add  passthru update script
- Update packages to the latest stable version

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 118367` at 538f4509 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages skipped due to time constraints:</summary>
  <ul>
    <li>cinnamon.cinnamon-screensaver</li>
    <li>mate.caja-extensions</li>
    <li>mate.mate-utils</li>
    <li>plata-theme</li>
  </ul>
</details>
<details>
  <summary>38 packages built successfully:</summary>
  <ul>
    <li>blueberry</li>
    <li>cinnamon.cinnamon-common</li>
    <li>cinnamon.cinnamon-session</li>
    <li>cinnamon.cjs</li>
    <li>cinnamon.nemo</li>
    <li>cinnamon.warpinator</li>
    <li>cinnamon.xapps</li>
    <li>cinnamon.xviewer</li>
    <li>mate.atril</li>
    <li>mate.caja</li>
    <li>mate.engrampa</li>
    <li>mate.eom</li>
    <li>mate.marco</li>
    <li>mate.mate-applets</li>
    <li>mate.mate-calc</li>
    <li>mate.mate-control-center</li>
    <li>mate.mate-indicator-applet</li>
    <li>mate.mate-media</li>
    <li>mate.mate-netbook</li>
    <li>mate.mate-notification-daemon</li>
    <li>mate.mate-panel</li>
    <li>mate.mate-power-manager</li>
    <li>mate.mate-screensaver</li>
    <li>mate.mate-sensors-applet</li>
    <li>mate.mate-session-manager</li>
    <li>mate.mate-settings-daemon</li>
    <li>mate.mate-system-monitor</li>
    <li>mate.mate-themes</li>
    <li>mate.mate-tweak</li>
    <li>mate.mate-user-share</li>
    <li>mate.mozo</li>
    <li>mate.pluma</li>
    <li>mate.python-caja</li>
    <li>python38Packages.xapp</li>
    <li>python39Packages.xapp</li>
    <li>xed-editor</li>
    <li>xmonad_log_applet_mate</li>
    <li>xplayer</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/desktops/mate/mate-tweak/default.nix:84:0:
```
   |
84 |     description = "Tweak tool for the MATE Desktop";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/desktops/mate/marco/default.nix [hunk 1]
```diff
@@ -1,12 +1,12 @@
-{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, libxml2, libcanberra-gtk3, libgtop, libstartup_notification, gnome3, gtk3, mate-settings-daemon, wrapGAppsHook }:
+{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, libxml2, libcanberra-gtk3, libgtop, libstartup_notification, gnome3, gtk3, mate-settings-daemon, wrapGAppsHook, mateUpdateScript }:
```
Comment (SuperSandro2000) [inline pkgs/desktops/mate/marco/default.nix:1]:
```suggestion
{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, libxml2, libcanberra-gtk3, libgtop
, libstartup_notification, gnome3, gtk3, mate-settings-daemon, wrapGAppsHook, mateUpdateScript }:
```

File: pkgs/desktops/mate/mate-applets/default.nix [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, gnome3, glib, gtk3, gtksourceview3, libwnck3, libgtop, libxml2, libnotify, polkit, upower, wirelesstools, mate, hicolor-icon-theme, wrapGAppsHook }:
+{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, gnome3, glib, gtk3, gtksourceview3, libwnck3, libgtop, libxml2, libnotify, polkit, upower, wirelesstools, mate, hicolor-icon-theme, wrapGAppsHook, mateUpdateScript }:
```
Comment (SuperSandro2000) [inline pkgs/desktops/mate/mate-applets/default.nix:1]:
```suggestion
{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, gnome3, glib, gtk3, gtksourceview3, libwnck3
, libgtop, libxml2, libnotify, polkit, upower, wirelesstools, mate, hicolor-icon-theme, wrapGAppsHook
, mateUpdateScript }:
```

File: pkgs/desktops/mate/mate-control-center/default.nix [hunk 3]
```diff
@@ -1,15 +1,16 @@
 { lib, stdenv, fetchurl, pkg-config, gettext, itstool, libxml2, dbus-glib,
   libxklavier, libcanberra-gtk3, librsvg, libappindicator-gtk3,
```
Comment (SuperSandro2000) [inline pkgs/desktops/mate/mate-control-center/default.nix:2]:
```suggestion
{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, libxml2, dbus-glib
, libxklavier, libcanberra-gtk3, librsvg, libappindicator-gtk3
```

File: pkgs/desktops/mate/mate-control-center/default.nix [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
-  desktop-file-utils, dconf, gtk3, polkit, mate, hicolor-icon-theme, wrapGAppsHook
+  desktop-file-utils, dconf, gtk3, polkit, mate, hicolor-icon-theme, wrapGAppsHook,
+  mateUpdateScript
```
Comment (SuperSandro2000) [inline pkgs/desktops/mate/mate-control-center/default.nix:4]:
```suggestion
, desktop-file-utils, dconf, gtk3, polkit, mate, hicolor-icon-theme, wrapGAppsHook
, mateUpdateScript
```

File: pkgs/desktops/mate/mate-sensors-applet/default.nix [hunk 5]
```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, gtk3, libxml2, libxslt, libatasmart, libnotify, lm_sensors, mate, hicolor-icon-theme, wrapGAppsHook }:
+{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, gtk3, libxml2, libxslt, libatasmart, libnotify, lm_sensors, mate, hicolor-icon-theme, wrapGAppsHook, mateUpdateScript }:
```
Comment (SuperSandro2000) [inline pkgs/desktops/mate/mate-sensors-applet/default.nix:1]:
```suggestion
{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, gtk3, libxml2, libxslt, libatasmart, libnotify
, lm_sensors, mate, hicolor-icon-theme, wrapGAppsHook, mateUpdateScript }:
```

File: pkgs/desktops/mate/mate-user-share/default.nix [hunk 6]
```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, gtk3, dbus-glib, libnotify, libxml2, libcanberra-gtk3, mod_dnssd, apacheHttpd, hicolor-icon-theme, mate, wrapGAppsHook }:
+{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, gtk3, dbus-glib, libnotify, libxml2, libcanberra-gtk3, mod_dnssd, apacheHttpd, hicolor-icon-theme, mate, wrapGAppsHook, mateUpdateScript }:
```
Comment (SuperSandro2000) [inline pkgs/desktops/mate/mate-user-share/default.nix:1]:
```suggestion
{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, gtk3, dbus-glib, libnotify, libxml2
, libcanberra-gtk3, mod_dnssd, apacheHttpd, hicolor-icon-theme, mate, wrapGAppsHook, mateUpdateScript }:
```

File: pkgs/desktops/mate/mate-utils/default.nix [hunk 7]
```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, glib, gtk3, libxml2, libgtop, libcanberra-gtk3, inkscape, udisks2, mate, hicolor-icon-theme, wrapGAppsHook }:
+{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, glib, gtk3, libxml2, libgtop, libcanberra-gtk3, inkscape, udisks2, mate, hicolor-icon-theme, wrapGAppsHook, mateUpdateScript }:
```
Comment (SuperSandro2000) [inline pkgs/desktops/mate/mate-utils/default.nix:1]:
```suggestion
{ lib, stdenv, fetchurl, pkg-config, gettext, itstool, glib, gtk3, libxml2, libgtop, libcanberra-gtk3
, inkscape, udisks2, mate, hicolor-icon-theme, wrapGAppsHook, mateUpdateScript }:
```

File: pkgs/desktops/mate/pluma/default.nix [hunk 8]
```diff
@@ -1,12 +1,12 @@
-{ lib, stdenv, fetchurl, pkg-config, gettext, perl, itstool, isocodes, enchant, libxml2, python3, gnome3, gtksourceview3, libpeas, mate, wrapGAppsHook }:
+{ lib, stdenv, fetchurl, pkg-config, gettext, perl, itstool, isocodes, enchant, libxml2, python3, gnome3, gtksourceview3, libpeas, mate, wrapGAppsHook, mateUpdateScript }:
```
Comment (SuperSandro2000) [inline pkgs/desktops/mate/pluma/default.nix:1]:
```suggestion
{ lib, stdenv, fetchurl, pkg-config, gettext, perl, itstool, isocodes, enchant, libxml2, python3
, gnome3, gtksourceview3, libpeas, mate, wrapGAppsHook, mateUpdateScript }:
```

**Review by SuperSandro2000 — COMMENTED** [review]
I added some line wrappings suggestions to keep the GitHub UI from scrolling.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 118367` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
45 packages built:
</summary>
<ul>
<li>blueberry</li>
<li>cinnamon.cinnamon-common</li>
<li>cinnamon.cinnamon-gsettings-overrides</li>
<li>cinnamon.cinnamon-screensaver</li>
<li>cinnamon.cinnamon-session</li>
<li>cinnamon.cjs</li>
<li>cinnamon.nemo</li>
<li>cinnamon.warpinator</li>
<li>cinnamon.xapps</li>
<li>cinnamon.xviewer</li>
<li>mate.atril</li>
<li>mate.caja</li>
<li>mate.caja-dropbox</li>
<li>mate.caja-extensions</li>
<li>mate.caja-with-extensions</li>
<li>mate.engrampa</li>
<li>mate.eom</li>
<li>mate.marco</li>
<li>mate.mate-applets</li>
<li>mate.mate-calc</li>
<li>mate.mate-control-center</li>
<li>mate.mate-indicator-applet</li>
<li>mate.mate-media</li>
<li>mate.mate-netbook</li>
<li>mate.mate-notification-daemon</li>
<li>mate.mate-panel</li>
<li>mate.mate-power-manager</li>
<li>mate.mate-screensaver</li>
<li>mate.mate-sensors-applet</li>
<li>mate.mate-session-manager</li>
<li>mate.mate-settings-daemon</li>
<li>mate.mate-system-monitor</li>
<li>mate.mate-themes</li>
<li>mate.mate-tweak</li>
<li>mate.mate-user-share</li>
<li>mate.mate-utils</li>
<li>mate.mozo</li>
<li>mate.pluma</li>
<li>mate.python-caja</li>
<li>plata-theme</li>
<li>python38Packages.xapp</li>
<li>python39Packages.xapp</li>
<li>xed-editor</li>
<li>xmonad_log_applet_mate</li>
<li>xplayer</li>
</ul>
</details>

**Comment by romildo** [comment]
This is working well for me. I think it should be merged.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 118367` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
45 packages built:
</summary>
<ul>
<li>blueberry</li>
<li>cinnamon.cinnamon-common</li>
<li>cinnamon.cinnamon-gsettings-overrides</li>
<li>cinnamon.cinnamon-screensaver</li>
<li>cinnamon.cinnamon-session</li>
<li>cinnamon.cjs</li>
<li>cinnamon.nemo</li>
<li>cinnamon.warpinator</li>
<li>cinnamon.xapps</li>
<li>cinnamon.xviewer</li>
<li>mate.atril</li>
<li>mate.caja</li>
<li>mate.caja-dropbox</li>
<li>mate.caja-extensions</li>
<li>mate.caja-with-extensions</li>
<li>mate.engrampa</li>
<li>mate.eom</li>
<li>mate.marco</li>
<li>mate.mate-applets</li>
<li>mate.mate-calc</li>
<li>mate.mate-control-center</li>
<li>mate.mate-indicator-applet</li>
<li>mate.mate-media</li>
<li>mate.mate-netbook</li>
<li>mate.mate-notification-daemon</li>
<li>mate.mate-panel</li>
<li>mate.mate-power-manager</li>
<li>mate.mate-screensaver</li>
<li>mate.mate-sensors-applet</li>
<li>mate.mate-session-manager</li>
<li>mate.mate-settings-daemon</li>
<li>mate.mate-system-monitor</li>
<li>mate.mate-themes</li>
<li>mate.mate-tweak</li>
<li>mate.mate-user-share</li>
<li>mate.mate-utils</li>
<li>mate.mozo</li>
<li>mate.pluma</li>
<li>mate.python-caja</li>
<li>plata-theme</li>
<li>python38Packages.xapp</li>
<li>python39Packages.xapp</li>
<li>xed-editor</li>
<li>xmonad_log_applet_mate</li>
<li>xplayer</li>
</ul>
</details>

**Review by SuperSandro2000 — APPROVED** [review]
diff LGTM

**Comment by ilya-fedin** [comment]
Am I the one who experience such a bug after upgrade?
изображение

**Comment by romildo** [comment]
> Am I the one who experience such a bug after upgrade?
> изображение

I am also experiencing problems with the icons on the notification area of the mate panel.

I have verified on my system that the issue persists even after reverting this PR. Therefore I think the issue was not introduced by this PR.

**Comment by ilya-fedin** [comment]
Only xembed icons seem to be affected, SNI ones are OK

**Comment by romildo** [comment]
@ilya-fedin would you open a new issue for this?

**Comment by ilya-fedin** [comment]
#122053

**Pull request merged**